resources/sass/pages/_cart.scss

```diff
@@ -0,0 +1,212 @@
+        &--add {
+            margin-right: 25px;
+            margin-left: 25px;
+            height: 25px;
+            width: 25px;
+            background: $color_red;
+            text-align: center;
+            color: $color_white;
+            border-radius: 100%;
+            display: inline-flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            font-size: 1rem;
+            font-weight: bold;
+            text-decoration: none;
+            font-family: sans-serif;
+        }
+        &--total {
+            margin-right: 50px;
+        }
+    }
+}
+
+.cart__content__price {
+    margin-top: 16px;
+    width: 100%;
+    padding: 12px;
+    display: flex;
+    align-items: center;
+    background: $color_white;
+}
+
+.button-small-checkout {
+    border: $color_red;
+    font-size: 100%;
+    padding: 10px;
+    background: $color_red;
+    color: $color_white;
+    width: 100%;
+    text-align: left;
+}
+
+.button-price {
+    float: right;
+}
+
+.checkout-box {
+    margin-top: 16px;
+    width: 100%;
+    padding: 12px;
+    background: $color_white;
+    .checkout-text {
+        text-align: center;
+    }
+}
+
+.cart-modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 31%;
+    top: 0;
+    width: 100%;
+    height: 87%;
+    top: 12%;
+    &-content {
+        position: absolute;
+        top: 0;
+        left: 0%;
+        bottom: 0;
+        background: #eeeeee;
+        margin-left: 35%;
+        margin-right: 35%;
+        border: 1px solid $color_black;
+    }
+    &-header {
+        padding: 2px 16px;
+        background-color: $color_black;
+        color: $color_white;
+        height: 60px;
+        font-family: $primary_font;
+        .close {
+            color: $color_white;
+            float: right;
+            font-size: 28px;
+            font-weight: bold;
+            &:hover,
+            &:focus {
+                color: $color_black;
+                text-decoration: none;
+                cursor: pointer;
+            }
+        }
+    }
+    &-body {
+        font-family: $primary_font;
+        height: 80%;
+        overflow-y: scroll;
+        width: 500px;
+    }
+    &-footer {
+        padding: 2px 16px;
+        margin-top: 10px;
+        &-text {
+            background: $color_white;
+            padding: 5px;
+            width: 100%;
+            height: 35px;
+            border: 1px solid $color_red;
+            cursor: pointer;
+            box-shadow: 0 50px $color_red inset;
+            color: $color_white;
+            margin-bottom: 2%;
+            font-size: 16px;
+        }
+    }
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-family: $primary_font;
+    font-weight: 400;
+    margin: 10px 0;
+}
+
+.modal-view {
+    padding-bottom: 13px;
+}
+
+.blank-cart {
+    text-align: center;
+    vertical-align: 40%;
+}
+
+.blank-cart-text {
+    margin-top: 40%;
+}
+
+.font-size {
```

Comment:
Why you changing font-size in middle of the application should be done one globally for entire application